test/todoService.spec.js

```diff
@@ -0,0 +1,32 @@
+describe('Service: Todo', function() {
+
+  var service;
```

Comment:
I like this pattern - creating a service, but I'm not sure it's necessary when the service is essentially extracting most of the controller